### PR TITLE
Updates for Quick start guide

### PR DESCRIPTION
**Pull request by apinnick**
- Fixed title (quickstart > quick start)
- Moved installer procedure to a module

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.10/Katello 4.12
* [x] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)
* [x] Foreman 3.8/Katello 4.10
* [x] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* [x] Foreman 3.4/Katello 4.6 (EL8 only)
* [x] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* We do not accept PRs for Foreman older than 3.3.

### CODE REVIEW

File: guides/common/attributes-titles.adoc [hunk 1]
```diff
@@ -21,7 +21,7 @@
 :MonitoringDocTitle: Monitoring {Project} performance
 :PlanningDocTitle: Planning for {ProjectName}
 :ProvisioningDocTitle: Provisioning hosts
-:QuickstartDocTitle: Quickstart guide for {Project} on {install-on-os}
+:QuickstartDocTitle: Quick start guide for {Project} on {install-on-os}
```
Comment (maximiliankolb) [inline guides/common/attributes-titles.adoc:24]:
```suggestion
:QuickStartDocTitle: Quick start guide for {Project} on {install-on-os}
```

To make it consistent how we go from title to attribute name, we should capitalize the `s`.

File: guides/common/modules/proc_running-installer.adoc [hunk 2]
```diff
@@ -0,0 +1,17 @@
+[id="running-installer_{context}"]
+= Running the Installer
+
+The installation run is non-interactive, but the configuration can be customized by supplying any of the options listed in `foreman-installer --help`, or by running `foreman-installer -i` for interactive mode.
+More examples are given in the Installation Options section.
+The `-v` option disables the progress bar and displays all changes.
+
+.Procedure
+
```
Comment (maximiliankolb) [inline guides/common/modules/proc_running-installer.adoc:9]:
```suggestion
```

File: guides/common/modules/proc_running-installer.adoc [hunk 3]
```diff
@@ -0,0 +1,17 @@
+[id="running-installer_{context}"]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_running-installer.adoc:1]: Please ensure that the anchor, title, and file name match.

File: guides/common/modules/proc_running-installer.adoc [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+= Running the Installer
```
Comment (maximiliankolb) [inline guides/common/modules/proc_running-installer.adoc:2]:
```suggestion
= Running the {Project} installer
```

File: guides/common/modules/proc_running-installer.adoc [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+The installation run is non-interactive, but the configuration can be customized by supplying any of the options listed in `foreman-installer --help`, or by running `foreman-installer -i` for interactive mode.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_running-installer.adoc:4]:
```suggestion
The installation run is non-interactive, but the configuration can be customized by supplying any of the options listed in `{foreman-installer} --help`, or by running `{foreman-installer} -i` for interactive mode.
```

File: guides/common/modules/proc_running-installer.adoc [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+More examples are given in the Installation Options section.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_running-installer.adoc:5]: I assume "Installation Options" refers to stdout from "foreman-installer --help". Does it need some emphasis then?

File: guides/common/modules/proc_running-installer.adoc [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+.Procedure
+
+* Run the following command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {installer-scenario}
+----
+
+When the installer has completed, details are printed about where to find Foreman and the Smart Proxy.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_running-installer.adoc:17]:
```suggestion
When the installer has completed, details are printed about where to find {ProjectServer} and {SmartProxy}.
```

I am also unsure about the wording "are printed".

**Review by maximiliankolb — CHANGES_REQUESTED** [review]

File: guides/common/modules/proc_running-installer.adoc (hunk 3 above)
Comment (apinnick) [reply guides/common/modules/proc_running-installer.adoc:1]: Our d/s modular docs reference says not to include the prefix in the anchor ID but I do not mind including it for the sake of consistency with the other modules. Will do.

File: guides/common/modules/proc_running-installer.adoc (hunk 7 above)
Comment (apinnick) [reply guides/common/modules/proc_running-installer.adoc:16]: Agree, re: "are printed". Will check.

Comment (apinnick) [reply guides/common/modules/proc_running-installer.adoc]: I replaced the last sentence with the text that is used in the d/s installation guide.

File: guides/common/modules/proc_running-installer.adoc [hunk 8]
```diff
@@ -0,0 +1,16 @@
+[id="proc_running-installer_{context}"]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_running-installer.adoc:1]:
```suggestion
[id="running-the-{project-context}-installer_{context}"]
```

**Review by maximiliankolb — CHANGES_REQUESTED** [review]
one minor comment, rest LGTM

**Review by maximiliankolb — APPROVED** [review]
LGTM

**Comment by maximiliankolb** [comment]
Merged to "master". There are merge conflicts when cherry-picking to 3.10. Please open a new PR against 3.10 or leave it be; both should be fine; your call @apinnick

**Pull request merged**